ci/publish-oci.sh

```diff
@@ -36,38 +40,62 @@ on_exit() {
 trap on_exit EXIT
 
 gen_manifest() {
-local dest=$1
-info "Generate manifest...\t\t"
+local component=$1
+if [[ "${component}" == "damlc" ]]; then
 echo '
 apiVersion: digitalasset.com/v1
 kind: Component
 spec:
   commands:
-  - path: "damlc"
+  - path: damlc
     name: damlc
-    desc: "damlc - Compiler and IDE backend for the Daml programming language"
-' > "${dest}" && info_done
+    desc: "Compiler and IDE backend for the Daml programming language"
+'
+elif [[ "${component}" == "daml-script" ]]; then
+echo '
+apiVersion: digitalasset.com/v1
+kind: Component
+spec:
+  jar-commands:
+    - path: daml-script-'${RELEASE_TAG}'.jar
```

Comment:
```suggestion
    - path: daml-script-'"${RELEASE_TAG}"'.jar
```
You can double-quote the RELEASE_TAG variable the same way you do further down in the script, e.g. on line 95.